agents/lakeshore372/ls372_configfiles.yaml

```diff
@@ -0,0 +1,24 @@
+LSASIM:
```

Comment:
So I think rather than including a file among the Agent code we should provide an example in the documentation, specifically on the page [docs/agents/lakeshore372.rst](https://github.com/simonsobs/socs/blob/develop/docs/agents/lakeshore372.rst).

Since labs use docker for this installation, they don't necessarily have a copy of the repo even.